Detectors/MUON/MCH/Simulation/data/simcuts.dat

```diff
@@ -0,0 +1,27 @@
+* MUON
```

Comment:
sorry for the stupid question, but are the simcuts.dat files meant to describe all materials or just the materials with non-defaults cuts ? 
In the latter case, it would be nice to add at the beginning of the file a short comment stating briefly what (and why) is changed.

Reply:
@aphecetche : I think for now this is just a copy-over from AliRoot. The precise reasoning might not be available at this moment. And yes, this is listing only non-default cuts so not every material has to be listed.

Reply:
I am going to fix the PR myself.